tests/e2e/e2e_query_test.go

```diff
@@ -0,0 +1,244 @@
+package e2e
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govv1beta1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
+	"github.com/cosmos/cosmos-sdk/x/group"
+	"github.com/ory/dockertest/v3/docker"
+)
+
+func (s *IntegrationTestSuite) queryGovProposal(endpoint string, proposalId uint64) (govv1beta1.QueryProposalResponse, error) {
+	var emptyProp govv1beta1.QueryProposalResponse
+
+	path := fmt.Sprintf("%s/cosmos/gov/v1beta1/proposals/%d", endpoint, proposalId)
+	resp, err := http.Get(path)
+	if err != nil {
+		s.T().Logf("This is the err: %s", err.Error())
```

Comment:
Should the test fail here?

Reply:
hmm good question...
looks like the next line contains the empty proposal response and an error.
these could be combined or the first check could be removed.